freqtrade/freqtradebot.py

```diff
@@ -95,6 +96,10 @@ def __init__(self, config: Dict[str, Any]) -> None:
         self.edge = Edge(self.config, self.exchange, self.strategy) if \
             self.config.get('edge', {}).get('enabled', False) else None
 
+        # Init ExternalMessageConsumer if enabled
+        self.emc = ExternalMessageConsumer(self.config, self.dataprovider) if \
```

Comment:
I guess this means only the follower should have this dict in the config?

Reply:
Yes